app/views/admin/users/index/_controls.html.erb

```diff
@@ -1,6 +1,36 @@
 <%= javascript_packs_with_chunks_tag "admin/users/controls", defer: true %>
-<% if params[:controller] == "admin/users" %>
```

Comment:
We didn't need this, since we only use this partial on the member index page 🤷‍♀️  I think perhaps we missed removing it in a refactor at some point